### PR TITLE
Release 1.0.2

### PR DESCRIPTION
**Pull request by benib**

### CODE REVIEW

File: slippyMap/resources/LeafletMap.js [hunk 1]
```diff
@@ -138,7 +137,7 @@ export default class LeafletMap {
     this.disableInteractions(this.map);
 
     try {
-      this.setAspectRatio(w, h);
+      this.setAspectRatio(w, h, false);
```
Comment (fuenkchen) [inline slippyMap/resources/LeafletMap.js]: is this 'false' parameter used somewhere - which I do not see?

File: slippyMap/resources/LeafletMap.js [hunk 2]
```diff
@@ -248,15 +251,15 @@ export default class LeafletMap {
...
+        this.map[moveFunctions.bounds](this.getBoundsWithMargin(featureGroupBounds));
       } else if (zoomLevel !== undefined) {
         // default zoom level when only one feature is 9
         if (zoomLevel === -1) {
           zoomLevel = 9;
         }
 
-        if (this.featureGroup.getBounds() && this.featureGroup.getBounds().getCenter()) {
-          this.map[moveFunctions.center](this.featureGroup.getBounds().getCenter(), zoomLevel);
+        if (featureGroupBounds && featureGroupCenter) {
```
Comment (fuenkchen) [inline slippyMap/resources/LeafletMap.js]: the check for featureGroupBounds is not needed here.

File: slippyMap/resources/LeafletMap.js [hunk 3]
```diff
@@ -238,7 +236,12 @@ export default class LeafletMap {
         center: 'flyTo'
       };
     }
-    if (this.featureGroup) {
+    const featureGroupBounds = this.featureGroup.getBounds();
+    let featureGroupCenter;
+    if (featureGroupBounds && featureGroupBounds.isValid()) {
+      featureGroupCenter = featureGroupBounds.getCenter();
+    }
+    if (this.featureGroup && featureGroupBounds && featureGroupCenter) {
```
Comment (fuenkchen) [inline slippyMap/resources/LeafletMap.js]: check for featureGroup is not needed, isn't it? it always is at least an empty featureGroup object.

**Review by fuenkchen — CHANGES_REQUESTED** [review]

**Comment by benib** [comment]
@fuenkchen thanks. corrected everything :-)

**Review by fuenkchen — APPROVED** [review]
👍

**Pull request merged**